src/analyses/variable-sensitivity/abstract_object.h

```diff
@@ -97,7 +99,7 @@ class abstract_objectt
 
   abstract_object_pointert clone() const
   {
-    return mutable_clone();
+    return abstract_object_pointert(mutable_clone());
```

Comment:
Is it necessary to convert this here? Is the return value of `mutable_clone` not automatically converted to `abstract_object_pointert`?